collectors/rds/describeOptionGroups.js

```diff
@@ -0,0 +1,29 @@
+var AWS = require('aws-sdk');
+var async = require('async');
+const _ = require("lodash");
```

Comment:
This is a heavy dependency for the smaller functionality below, can we re-write that using more native JS?

Reply:
@matthewdfuller - What makes this a heavy dependency?  The use of lodash?  Can you please provide more details so that this gets reworked inline with what is expected?

Reply:
@philcorn - we would need to import the entire lodash module for the project just to do `_.find` in this one script. It's nothing against lodash specifically, but mostly as an internal practice we try to minimize third-party libraries where possible in favor of native JavaScript/Node. It helps reduce security concerns that often arise from the web of dependencies that Node modules have.